yamale/validators/constraints.py

```diff
@@ -34,11 +34,23 @@ def get_kwarg(self, kwargs, key, kwtype):
         try:  # Try to convert value
             # Is this value one of the datetime types?
             if kwtype == datetime.date:
-                time = datetime.datetime.strptime(value, '%Y-%m-%d')
+                # Get custom format if used
+                if kwargs['format']: 
```

Comment:
Looks like the build complained about this line. 

https://travis-ci.org/github/23andMe/Yamale/jobs/753359620#L245

Reply:
I think I fixed this. I changed this from ```if kwargs['format']:``` to ```if 'format' in kwargs:``` so that we don't get a ```KeyError``` if the format arg isn't used.